c2corg_ui/templates/helpers/view.html

```diff
@@ -255,10 +270,17 @@ <h3 class="heading show-phone outings" ng-click="mainCtrl.animateHeaderIcon($eve
             </a>
           </div>
         % endif
-        <div class="list-item outings" ng-repeat="doc in detailsCtrl.documentService.document.associations.recent_outings.documents">
+        <div class="list-item outings" ng-repeat="doc in detailsCtrl.documentService.document.associations.${'recent_outings.documents' if isRecentOutings else 'outings'}">
           <app-card app-card-doc="doc"></app-card>
+          % if not isRecentOutings and document['doctype'] == 'images':
+            <%
+                options = {"imageType": document["image_type"], "imageCreator": document["creator"]["user_id"]}
+            %>
+            <app-delete-association child-doctype="o" parent-id="${document['document_id']}" child-id="doc.document_id"
+              ng-if="userCtrl.hasEditRights('images', ${options})"></app-delete-association>
```

Comment:
Not sure this test sufficient. But anyway there is probably some work to do to fix the association removal permissions, see https://github.com/c2corg/v6_ui/issues/836